packages/common/src/util.js

```diff
@@ -108,6 +108,30 @@ util.ApiError = function(errorBody) {
 };
 
 /**
+ * Custom error type for partial errors returned from the API.
+ *
+ * @param {object} b - Error object.
+ */
+var PartialFailureError = createErrorClass('PartialFailureError', function(b) {
+  var errorObject = b;
+
+  this.errors = errorObject.errors;
+  this.response = errorObject.response;
+
+  var defaultErrorMessage = 'A failure occurred during this request.';
+  this.message = errorObject.message || defaultErrorMessage;
+});
+
+/**
+ * Wrap the PartialFailureError constructor so context isn't lost.
+ *
+ * @param {object} errorBody - Error object.
+ */
+util.PartialFailureError = function(errorBody) {
```

Comment:
Sure can't, there are a couple errors floating around BigTable that make use of `createErrorClass` and I've never seen/heard any issues with them. `ApiError` predates our use of `createErrorClass`, is it possible that was an old comment that never got cleaned up?